src/metrics/metadata/metadata.go

```diff
@@ -94,7 +94,7 @@ type PipelineMetadata struct {
 	GraphitePrefix [][]byte `json:"graphitePrefix,omitempty"`
 
 	// ResendEnabled is true if the Pipeline supports resending aggregate values after the initial flush.
-	ResendEnabled bool
+	ResendEnabled bool `json:"resendEnabled,omitempty"`
```

Comment:
ah thanks! I don't think it should be too late, I don't think we use json marshaling